# Removes the "Get help with using Chromium" button, which triggers a bug in another patch when domain substitution and certain other conditions are met.
# This is a workaround for a case of Issue #42. See Issue #42 for more details on this bug.

--- a/chrome/browser/resources/help/help_content.html
+++ b/chrome/browser/resources/help/help_content.html
@@ -9,7 +9,6 @@
     </div>
   </div>
   <div id="help-container">
-    <button id="get-help" i18n-content="getHelpWithChrome"></button>
 <if expr="_google_chrome">
     <button id="report-issue" i18n-content="reportAnIssue"></button>
 </if>
--- a/chrome/browser/resources/help/help_page.js
+++ b/chrome/browser/resources/help/help_page.js
@@ -90,10 +90,6 @@ cr.define('help', function() {
       var productTOS = $('product-tos');
       if (productTOS)
         productTOS.innerHTML = loadTimeData.getString('productTOS');
-
-      $('get-help').onclick = function() {
-        chrome.send('openHelpPage');
-      };
 <if expr="_google_chrome">
       $('report-issue').onclick = function() {
         chrome.send('openFeedbackDialog');
